P5 247 rewrite cnn (#164)
aau-hcl-p5/main: report/3.Theory/ObjectRecognition.tex

\subsection{Convolutional Neural Network}
\label{sec:cnn}
Convolutional neural networks (CNN) are often used for image classification, and is a popular type of neural network in regards to object detection and image processing.

> In challenges such as LSVCR\footnote{Imagenet Large Scale Visual Recognition Challenge} the majority of the entries are done using CNN \cite{ILSVRC_Results}. 

Reviewer: ```suggestion
In challenges such as Imagenet Large Scale Visual Recognition Challenge (LSVCR), the majority of the entries are done using CNN \cite{ILSVRC_Results}. 
```

The special thing about a CNN is the addition of convolutional layers.

> The convolutional layers will add convolutional filters, which main purpose is to act as a kind of feature detector.

Reviewer: ```suggestion
The convolutional layers will add convolutional filters, that has the main purpose of acting as a kind of feature detector.
```

@nillas12 red mig min edit hører til i skraldespadnen

The filters works by combining $n \times n$ inputs into a single output.
The output can resemble some feature found within the convolutional layer.

> \figur{0.8}{images/cnn}{Convolution layer illustration\cite{cnnguide}}{fig:cnn}

Reviewer: figurer skal refererers til hvis de skal med
